CONTRIBUTE.md

```diff
@@ -0,0 +1,39 @@
+# Contribution guide
+
+This document summarizes how work is organized and structured using Git to lay out the common guidelines for contributions to this project.
+
+## The project's structure in repositories
+
+The project is split up in two distributed repositories – one for the frontend and backend part of the application. This is done to completely separate the two subsystems from each other, and let the repositories be managed by their own version control setup to control releases and such independently.
+
+The repositories, hosted on GitHub, support an intended centralized workflow in which each one of them acts as a shared repository that all contributors contribute their work to.
+
+Link to frontend repository: https://github.com/caspar-itu/minitwit-react-frontend
+Link to backend repository: http://github.com/emilbartholdy
```

Comment:
Please update to
https://github.com/DevOps-Drengene/minitwit-react-frontend
https://github.com/DevOps-Drengene/minitwit-backend

Reply:
Thanks, it is done now!